src/Inventory.zig

```diff
@@ -896,8 +896,9 @@ pub const Command = struct { // MARK: Command
 			const typ: Inventory.Type = @enumFromInt(data[12]);
 			const sourceType: SourceType = @enumFromInt(data[13]);
 			const source: Source = switch(sourceType) {
-				.playerInventory => .{.playerInventory = {}},
+				.playerInventory => .{.playerInventory = std.math.maxInt(u32)},
 				.sharedTestingInventory => .{.sharedTestingInventory = {}},
+				.hand => .{.hand = std.math.maxInt(u32)},
```

Comment:
Please serialize these values here instead of just setting it to a random number. Otherwise it might fail in unexpected ways.